pkg/controller/core/workload_controller.go

```diff
@@ -104,11 +138,30 @@ func (r *WorkloadReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 			return ctrl.Result{}, client.IgnoreNotFound(err)
 		}
 	case admitted:
-		msg := fmt.Sprintf("Admitted by ClusterQueue %s", wl.Spec.Admission.ClusterQueue)
-		err := workload.UpdateStatusIfChanged(ctx, r.client, &wl, kueue.WorkloadAdmitted, metav1.ConditionTrue, "AdmissionByKueue", msg)
-		return ctrl.Result{}, client.IgnoreNotFound(err)
+		var err error
+		var requeueAfter time.Duration
+		resetAdmission := false
```

Comment:
cancelAdmission?

Reply:
Renamed, I like `cancel` in this context